README.md

```diff
@@ -1,41 +1,50 @@
 [![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue)
 
-# fairlearn
+# Fairlearn
+
+The Fairlearn project seeks to enable anyone involved in the development of artificial intelligence (AI) systems to assess their system's fairness and mitigate the observed unfairness. The Fairlearn repository contains a Python package and Jupyter notebooks with the examples of usage.
+
+ #### Fairlearn components ####
+ * Assessment dashboard: high-level and detailed views, for assessing which groups are negatively impacted. 
```

Comment:
Should metrics be broken out separately, or are we putting them under the dashboard?

Reply:
Good point. I think we can hold off on this one for a future PR.